build.libs.toml

```diff
@@ -0,0 +1,82 @@
+[metadata]
+
+[versions]
+# Android SDK versions
+compileSdk = "34"
+minSdk = "33"
+targetSdk = "34"
+
+# Test versions
+junit4 = "4.13.2"
+junit5 = "5.10.0" # careful, upgrading this can change a Cipher's IV size in tests!?
+mockk = "1.13.4" # newer versions require kotlin > 1.8.10
+espresso = "3.4.0"
+
+# Dependency versions below this are AOSP versions.
+# We use "strictly" to enforce the version cannot be overriden by transitive dependencies.
+# We need to enforce that the versions we use are the same as AOSP to ensure compatibility.
+
+# Kotlin versions
```

Comment:
The comment for this got lost, https://android.googlesource.com/platform/external/kotlinc/+/refs/tags/android-14.0.0_r1/build.txt

Reply:
Added it back.